### DISCUSSION TITLE
Sample Prometheus alerting rules

### DISCUSSION THREAD

**Discussion by azizzailani** [ClusterLabs/ha_cluster_exporter#261] [Q&A] [answered]
Do you have any prometheus alerting rules you could share as a starting point?

Thanks,

**Comment by azizzailani** [comment]
@stefanotorresi help

**Comment by stefanotorresi** [comment, marked as answer]
hey there, sorry about the very late reply.

You can find some examples here:
https://github.com/SUSE/ha-sap-terraform-deployments/blob/main/salt/monitoring_srv/prometheus/rules.yml

But bear in mind these have not been tested for a while, are not authoritative nor exhaustive in any way, shape or form.
